paasta_tools/deployd/common.py

```diff
@@ -30,11 +33,12 @@
     [
         "service",
         "instance",
+        "watcher",
+        "cluster",
```

Comment:
Good question. I added it because ServiceInstance takes cluster as a parameter, and I noticed it wasn't actually doing anything with it.

Looks like I can just drop the parameter -- turns out `ServiceInstance` took `cluster` so that it could look up the appropriate priority from soa-configs: https://github.com/Yelp/paasta/pull/2343/files#diff-0d5c7263f04c8928f403a53f529860cfL74

I think I may also just unify BaseServiceInstance / ServiceInstance; I think the only reason ServiceInstance existed was to have an __init__ that called `get_priority`.